package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
This is the file generated when using Node v16.
The CI action seems to have no issue with it.